### DISCUSSION TITLE
What's the base tag in the 404 page for?

### DISCUSSION THREAD

**Discussion by Steve0Greatness** [theforumhelpers/theforumhelpers.github.io#203] [Q&A] [answered]
I was working on changing part of the site, then I looked at the 404 page, and found that it didn't work. After looking into it, I can conclude that it's simply because of the base tag.
Why is it there? What purpose does it serve(I even commented it out... everything worked fine)?

**Comment by Accio1** [comment, marked as answer]
The base tag means that all relative URLS on that page will base off of the contents of the tag. For local development, you need to change it to how your filesystem is set up. For example, if your forumhelpers clone was at file:///C:/Users/USER/Downloads/theforumhelpers, then you would have to add that to the base tag so the relative URLs are based off of that path.

**Reply by Steve0Greatness** [reply]
but why is it there? There's no real purpose...

**Reply by Accio1** [reply]
If it is removed, the page breaks if there is a page not found with a slash at the end.

Eg. https://theforumhelpers.github.io/a would display the correct 404 page, but https://theforumhelpers.github.io/a/ would not. The base element allows the relative links in the file to point to the correct place. Believe me, it wouldn't have been added if it wasn't necessary.

**Reply by leahcimto** [reply]
(more info at #141)
